lib/rules/jsx-indent-props.js

```diff
@@ -160,8 +163,33 @@ module.exports = {
 
     return {
       JSXOpeningElement: function(node) {
-        const elementIndent = getNodeIndent(node);
-        checkNodesIndent(node.attributes, elementIndent + indentSize);
+        if (!node.attributes.length) {
+          return;
+        }
+        let propIndent;
+        if (indentSize === 'aligned') {
+          const firstPropNode = node.attributes[0];
+          // '<' and any space following it
+          const beforeNameLength = node.name.range[0] - node.range[0];
+          // the name itself
+          const nameLength = node.name.range[1] - node.name.range[0];
+          // any space between the name and the first attribute
+          const afterNameLength = Math.max(1, firstPropNode.loc.start.column - node.name.loc.end.column);
+          if (afterNameLength !== 1) {
+            // this is not easily fixable, so we just report another error for it.
```

Comment:
I went for the lazy option here - the default fixing logic in `report()` cannot handle the case where the property isn't the first thing in the line and `checkNodesIndent` would need further changes as well.

If you think having an automated fixer for this is important I can have another look though.

Reply:
Yes, I think it's very important to have an automated fixer for as many things as possible.

Reply:
OK I'll have another look at this